erizo_controller/erizoClient/src/Stream.js

```diff
@@ -280,25 +280,41 @@ Erizo.Stream = function (spec) {
         }
     };
 
+    that.muteAudio = function (audioActive, callback) {
+        if (that.room && that.room.p2p){
+            L.Logger.warning('muteAudio is not implemented in p2p streams');
+            callback ('error');
```

Comment:
Maybe we can send a message to the stream publisher to ask it to close the audio track. This way the feature will work in both scenarios.